rust/ql/src/queries/telemetry/ExtractorInformation.ql

```diff
@@ -41,6 +42,20 @@ predicate extractorDiagnostics(string key, int value) {
   )
 }
 
+predicate pathResolutionCompare(string key, int value) {
+  exists(string suffix |
+    PathResolutionCompare::summary(suffix, value) and
+    key = "Rust-analyzer path resolution comparison: " + suffix
```

Comment:
I guess I would expect the value to come after a `:`. Maybe `,` or some other symbol would work better?

Reply:
I would rather not change it, since we already have logic in DCA that depends on the exact wording of the string.